modules/ROOT/pages/_partials/flex-tls-cipher.adoc

```diff
@@ -0,0 +1,69 @@
+//tag::connectedSelect[]
+
+== Select Ciphers
+
+When you configure a TLS context, Secrets Manager applies default ciphers based on the TLS versions you select. In addition to the defaults, you can select other ciphers to use with the selected TLS version. Each TLS context can have multiple ciphers.
+
+To select ciphers:
+
+. Click *Ciphers* to see available ciphers. 
+. Select ciphers.
++
+If you select a *TLS Version* value of *TLS 1.3*, the default selection is not customizable.
+. Click *Save*
+
+//end::connectedSelect[]
+
+//tag::cipherSupport[]
+
+== TLS Support on Flex Gateway
+
+Flex can support a range of TLS Versions from TLS 1.1 to TLS 1.3, and you can also customize some of the ciphers to support.
+
+You cannot customize the list of TLS 1.3 Ciphers. If you support TLS 1.3, you must support the TLS 1.3 default ciphers.
+
+If you support TLS 1.2, the TLS 1.2 default Ciphers are selected. However, unlike TLS 1.3, you can customize different TLS 1.2 ciphers.
+
+There are no default ciphers for TLS 1.1. If you choose to support TLS 1.1, you must select the ciphers you want to support.
+
+IMPORTANT: Ensure that your API upstream supports the selected ciphers and versions.
+
+//end::cipherSupport[]
+
+//tag::ciphers[]
+
+=== Flex Gateway Supported Ciphers
+
+Flex Gateway supports the following TLS Ciphers in connected and local mode:
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+|===
+| Cipher | TLS Version | Default | Advice
+| TLS_AES_128_GCM_SHA256 | 1.3 | Yes | Secure
```

Comment:
Note my comment above. TLS 1.3 ciphers are not shown in the UI. I would treat 1.3 in a separate section and highlight that those are the TLS 1.3 ciphers available with Flex version 1.3 and 1.4. 
Alternatively there could be a link to a Flex release notes document where we could add the ciphers supported.